ENIGMAsystem/SHELL/Platforms/Win32/WINDOWSwindow.cpp

```diff
@@ -39,108 +39,113 @@ static int displayInitialResolutionWidth = 0, displayInitialResolutionHeight = 0
+    int screen_height = GetSystemMetrics(SM_CYSCREEN);
+    enigma::windowX = (screen_width - enigma::regionWidth)/2;
+    enigma::windowY = (screen_height - enigma::regionHeight)/2;
+  }
 
-    void clampwindow()
-    {
-        RECT c;
-        c.left = enigma::windowX; c.top = enigma::windowY; c.right = enigma::windowX + enigma::windowWidth; c.bottom = enigma::windowY + enigma::windowHeight;
-        AdjustWindowRect(&c, getwindowstyle(), false);
-        SetWindowPos(enigma::hWnd, HWND_TOP, c.left, c.top, c.right-c.left, c.bottom-c.top, SWP_NOZORDER|SWP_FRAMECHANGED);
-    }
+  void clampwindow()
+  {
+    RECT c;
+    c.left = enigma::windowX; c.top = enigma::windowY; c.right = enigma::windowX + enigma::windowWidth; c.bottom = enigma::windowY + enigma::windowHeight;
+    AdjustWindowRect(&c, GetWindowLongPtr(enigma::hWnd, GWL_EXSTYLE), false);
+    SetWindowPos(enigma::hWnd, HWND_TOP, c.left, c.top, c.right-c.left, c.bottom-c.top, SWP_NOZORDER|SWP_FRAMECHANGED);
+  }
 
-    void setwindowstyle()
-    {
-        SetWindowLongPtr(enigma::hWnd,GWL_STYLE,getwindowstyle());
-        clampwindow();
-    }
+  void setwindowsize()
+  {
+      if (!regionWidth)
+	  return;
 
-    void setwindowsize()
-    {
-        if (!regionWidth)
-            return;
-
-        int parWidth = isFullScreen?GetSystemMetrics(SM_CXSCREEN):windowWidth, parHeight = isFullScreen?GetSystemMetrics(SM_CYSCREEN):windowHeight;
-        if (viewScale > 0)  //Fixed Scale
-        {
-            double viewDouble = viewScale/100.0;
-            scaledWidth = regionWidth*viewDouble;
-            scaledHeight = regionHeight*viewDouble;
-        }
-        else if (viewScale == 0)  //Full Scale
-        {
-            scaledWidth = parWidth;
-            scaledHeight = parHeight;
-        }
-        else  //Keep Aspect Ratio
-        {
-            double fitWidth = parWidth/double(regionWidth), fitHeight = parHeight/double(regionHeight);
-            if (fitWidth < fitHeight)
-            {
-                scaledWidth = parWidth;
-                scaledHeight = regionHeight*fitWidth;
-            }
-            else
-            {
-                scaledWidth = regionWidth*fitHeight;
-                scaledHeight = parHeight;
-            }
-        }
-
-        if (!isFullScreen)
-        {
-            if (windowAdapt && viewScale > 0) // If the window is to be adapted and Fixed Scale
-            {
-                if (scaledWidth > windowWidth)
-                    windowWidth = scaledWidth;
-                if (scaledHeight > windowHeight)
-                    windowHeight = scaledHeight;
-            }
-             clampwindow();
-        } else {
-          SetWindowPos(hWnd, NULL, 0, 0, parWidth, parHeight, SWP_NOACTIVATE);
-        }
+      int parWidth = isFullScreen?GetSystemMetrics(SM_CXSCREEN):windowWidth, parHeight = isFullScreen?GetSystemMetrics(SM_CYSCREEN):windowHeight;
+      if (viewScale > 0)  //Fixed Scale
+      {
+	  double viewDouble = viewScale/100.0;
```

Comment:
This line has a tab followed by two spaces before `double` and so do the surrounding lines, please fix it.